Sources/mParticle-Appboy/MPKitAppboy.m

```diff
@@ -550,11 +551,16 @@ - (MPKitExecStatus *)routeCommerceEvent:(MPCommerceEvent *)commerceEvent {
                     [properties addEntriesFromDictionary:productDictionary];
                 }
                 
+                NSString *sanitizedProductName = product.sku;
```

Comment:
So looks like this logic is correct, but the setting from the server is poorly named (it's the opposite of what it says). Please add a comment above this line explaining that this behavior is intentional.